content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+---
+title: Automating operations on Data Services Cloud Console using Ansible Playbooks
```

Comment:
Hi there. A similar question in regards to the full name of the product. Isn't it HPE GreenLake Data Services Cloud Console? If that's the case, then you'll need to use that in the title (I know... it's hard because it makes it so long... I get it.)